official/recommendation/ncf_common.py

```diff
@@ -110,10 +110,10 @@ def parse_flags(flags_obj):
 
 
 def get_optimizer(params):
-  optimizer = tf.train.AdamOptimizer(
```

Comment:
fyi, i have heard there are slight numerical differences between the old and new adam optimizers. something to be aware of as you compare convergence etc.